server/server.js

```diff
@@ -14,12 +14,18 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 app.use(express.static(path.join(__dirname, '../client/dist')));
 
-app.get('/users', (req, res) => {
-  db.any('SELECT * FROM users WHERE id < $1', [10])
+app.get('/listings/:tagId', (req, res) => {
+  const listingData = {};
+  db.any('SELECT DISTINCT reviews.body, reviews.overall, users.firstname, users.imageurl FROM reviews JOIN users ON reviews.user_id = users.id WHERE reviews.listing_id = $1', [req.params.tagId])
     .then((data) => {
-      res.status(200).send({
-        message: data,
-      });
+      listingData.reviews = data;
+      return db.any('SELECT * from listings where id = $1', [req.params.tagId]);
+    })
+    .then((data) => {
+      listingData.overview = data;
+    })
+    .then(() => {
+      res.status(200).send(listingData);
     })
     .catch((error) => {
       res.status(200).send({
```

Comment:
consider putting in an error code here rather than 200

Reply:
I kinda like this setup for development, but it is definitely bad practice for production. I'll fix it later.